src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -235,8 +235,8 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   /// Dead nodes.
   absl::flat_hash_map<NodeID, std::shared_ptr<rpc::GcsNodeInfo>> dead_nodes_;
   /// The nodes are sorted according to the timestamp, and the oldest is at the head of
-  /// the list.
-  std::list<std::pair<NodeID, int64_t>> sorted_dead_node_list_;
```

Comment:
`std::list` should be **ONLY** used when insertion / deletion in the middle is a necessary operation.